src/config.rs

```diff
@@ -9,12 +9,21 @@ use serde::{Deserialize, Serialize};
 #[serde_inline_default]
 #[derive(Debug, Serialize, Deserialize)]
 pub struct Config {
-    #[serde(default)]
+    #[serde_inline_default(Config::default().aur_rm_args)]
     pub aur_rm_args: Vec<String>,
-    #[serde_inline_default(true)]
+    #[serde_inline_default(Config::default().flatpak_systemwide)]
     pub flatpak_systemwide: bool,
-    #[serde(default)]
-    pub hostnames: BTreeMap<String, Vec<String>>,
+    #[serde_inline_default(Config::default().hostname_groups)]
+    pub hostname_groups: BTreeMap<String, Vec<String>>,
+}
+impl Default for Config {
+    fn default() -> Self {
+        Config {
```

Comment:
We REALLY need a disabled backends feature. For pip atleast (for starters, not sure what all else).

Reply:
That seems fair, do pip and pipx clash the same way as yay and paru?

Reply:
yup

Reply:
Maybe we should just remove pip like you mentioned a while back

Reply:
It gets worse with pip. It doesn't just clash with pipx, it also clashes with whatever puts anything on its path.

Reply:
I've removed pip, I hear good things about the `uv` package manager for python, perhaps that might be another reasonable replacement, though it might also clash with `pipx`.